src/main/java/uk/gov/hmcts/cmc/claimstore/events/ccd/CoreCaseDataUploader.java

```diff
@@ -22,7 +23,16 @@ public CoreCaseDataUploader(CoreCaseDataService coreCaseDataService) {
     }
 
     @TransactionalEventListener
-    public void saveClaim(RepresentedClaimIssuedEvent event) {
+    public void saveRepresentedClaim(RepresentedClaimIssuedEvent event) {
```

Comment:
Is there any reason why these need to be split for caseworker/citizen? It's exactly the same code.